src/Flex.php

```diff
@@ -587,6 +587,20 @@ private function fetchRecipes(): array
             $name = $package->getNames()[0];
             $job = $operation->getJobType();
 
+            if (!empty($manifests[$name]['manifest']['conflict'])) {
+                $lockedRepository = $this->composer->getLocker()->getLockedRepository();
+
+                foreach ($manifests[$name]['manifest']['conflict'] as $conflictingPackageName => $constraint) {
+                    if ($conflictingPackage = $lockedRepository->findPackage($conflictingPackageName, $constraint)) {
+                        if ($this->io->isVerbose()) {
+                            $this->io->writeError(sprintf('    Recipe for %s conflicts with %s:%s which should not be %s', $name, $conflictingPackageName, $conflictingPackage->getPrettyVersion(), $constraint));
```

Comment:
Skipping recipe for %s: it conflicts with %s %s.

Reply:
Done.